pkg/graph/schema.graphql

```diff
@@ -1330,6 +1364,16 @@ type Query {
   systemIntake(id: UUID!): SystemIntake
   systems(after: String, first: Int!): SystemConnection
   currentUser: CurrentUser
+  cedarAuthorityToOperate(
```

Comment:
Assuming we get rid of the optional parameters, we'll want to make `cedarSystemID` non-nullable and remove the other parameters to this query.